main.py

```diff
@@ -2,17 +2,14 @@
 
 from kytos.core import KytosEvent, KytosNApp, log
 from kytos.core.helpers import listen_to
-
+from napps.kytos.of_l2ls import settings
 from pyof.foundation.network_types import Ethernet
-
 # OpenFlow structures that are common for both versions will be imported once.
```

Comment:
This comment was removed and should be added back here:
```
# OpenFlow structures that differ will be imported versionwise.
```

Reply:
The change have been completed.